### ISSUE TITLE
adding papers?

### ISSUE THREAD

**Issue by telephon** [toplap/awesome-livecoding#12]
I'm not sure how large the paper section should be.

but this is a good overview, I still think: http://drops.dagstuhl.de/opus/frontdoor.php?source_opus=4420

**Comment by lvm** [comment]
yes! originally i wanted to add papers but wasn't sure which ones to add.  
for the moment, i'll add this one in `to-read` and leave this issue open until we get some more papers and then will create a `papers` section.  
does that sounds good?

**Comment by telephon** [comment]
sounds good. 


if the section is extended it might be good if all papers have a 2-3 sentence description

**Comment by lvm** [comment]
Yes, totally. Do you have by chance a list of papers you would like to add? (i'm not really sure where to start looking)

**Comment by charlieroberts** [comment]
It'd be great if the list of papers on the main page could be kept small and curated to be papers providing general overviews of live coding practice as opposed to research on more specific topics (and I would certainly include the Dagstuhl summary in that group). Perhaps a second publication page could be added that breaks individual publications down by category?

Maybe that could be a strategy for videos as well... curate videos that are somehow "important" to live-coding on the main page, but then have a secondary page where anyone can add a live-coding video along with a summary.

**Comment by telephon** [comment]
the dagstuhl report is much broader in scope, i think, than many overviews of practice.But it's of course up to you as a curator. I feel a responsibility for keeping people aware that live coding is an open and conceptual field rather than a practical issue. This is often a little difficult, and I totally understand if others don't share this concern.

**Comment by telephon** [comment]
of course a separate page for focussed topics is good. In the end, the toplap publication list could be kept in sync with it

**Comment by lvm** [comment]
I like the approach suggested by @charlieroberts, like _a place to take off_ and then to go deeper down the rabbit hole if that's what you prefer.   
On the other hand, also understand  what you (@telephon) propose and probably would be a good idea to broaden the spectrum. Actually, there was a [discussion on Twitter](https://twitter.com/yaxu/status/831911724181757953) about this issue. My position on this matter is: _it's open for contributions_. I only happened to create the repo but I'd really like that this list to be community driven since some of the information available here was taken from Toplap itself or the LiveCode Slack (ie: _the/a community_).  
Yet still, if we start to add long chunks of text on the main page it might get difficult to navigate, so sub-pages with extra content would be the most sane route, I think. Similar to the [CONTRIB](../blob/master/CONTRIB.md) page.  
  
Does this makes sense?

**Comment by telephon** [comment]
yes!
